src/components/editor-toolbar/index.jsx

```diff
@@ -60,10 +60,13 @@ const EditorToolbar = ( { className } ) => {
 	}, [] );
 	const { setIsInserterOpened } = useDispatch( editorStore );
 
-	useModalize( isInserterOpened );
+	useModalize( isInserterOpened && ! enableNativeBlockInserter );
 	useModalize( isBlockInspectorShown );
 
-	useModalDialogState( isInserterOpened, 'block-inserter' );
+	useModalDialogState(
+		isInserterOpened && ! enableNativeBlockInserter,
+		'block-inserter'
+	);
```

Comment:
These marked content outside of the block inserter modal dialog "inert," and thus inaccessible by screen readers. This is unnecessary for the native inserter, as the native UI handles this itself. The WebView is inaccessible to screen readers when the native inserter is open.